### PR TITLE
Adding option to pass ACL parameter.

### PR DESCRIPTION
**Pull request by igneosaur**
Thought I might as well whack in a change here, hopefully this is what you had in mind for the feature. Addresses issue #11.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ function defaultKey (req, file, cb) {
   })
 }
 
+function defaultAcl (req, file, cb) {
+  setImmediate(function () { cb(null, 'private') })
```
Comment (LinusU) [inline index.js]: Could you make this `setImmediate(cb, null, 'private')` to avoid creating an anonymous function.

**Comment by LinusU** [comment]
One nit, otherwise it LGTM 👍 great job!

File: index.js (hunk 1 above)
Comment (LinusU) [inline index.js]: I realise now that the other call looks like this as well, so feel free to leave it and then I'll submit a PR to fix both of them later...

File: README.md [hunk 2]
```diff
@@ -36,6 +36,25 @@ app.post('/upload', upload.array('photos', 3), function(req, res, next) {
...
+    bucket: 'some-bucket',
+    acl: 'public-read',
+    key: function (req, file, cb) {
+      cb(null, Date.now().toString())
+    }
+  })
+})
+```
+
+Options: `private | public-read | public-read-write | aws-exec-read | authenticated-read | bucket-owner-read | bucket-owner-full-control`.
```
Comment (LinusU) [inline README.md]: Maybe make this a list, and also add in a description of each option?

    Options:

    - `private` - Can only be accessed with valid credentials
    - `public-read` - Can be seen by anyone that has the link
    ...

We can probably copy the description from amazons own documentation somewhere...

File: index.js (hunk 1 above)
Comment (igneosaur) [inline index.js]: Might as well fix it whilst we're here. I've made this change in the newest commits now.

File: README.md (hunk 2 above)
Comment (igneosaur) [inline README.md]: Ok, I've listed out all the options in a nice wee table now.

**Comment by igneosaur** [comment]
This is ready to roll 🚀

**Comment by igneosaur** [comment]
I've just merged in the changes from #29 and added the new `ACL` option to @thebergamo's new table.

This is ready to roll 🚀

**Comment by LinusU** [comment]
Looking great :raised_hands: LGTM

**Pull request merged**